packages/composites/por-indexer/src/PorInputAddress.ts

```diff
@@ -0,0 +1,6 @@
+export interface PorInputAddress {
```

Comment:
Should this go in `@chainlink/types`?

Reply:
probably not there, but the PoR composite would make sense to me